### DISCUSSION TITLE
Is there a guide to locally debug Apache Kvrocks in VS Code or other IDEs?

### DISCUSSION THREAD

**Discussion by tenghuanhe** [apache/kvrocks#1536] [Q&A] [answered]
Hello everyone,

I am new to Apache Rocks and would like to learn more about it by debugging the project locally using an IDE such as VS Code or Clion. However, I'm not sure how to set up the project for debugging in these IDEs.

I am hoping someone could provide guidance on how to set up the project for debugging in an IDE or point me in the direction of any existing resources that could be helpful.

Thank you very much for your time and assistance.

**Comment by git-hulk** [comment, marked as answer]
I'm using Clion, it can be as simple as:

1. git clone https://github.com/apache/kvrocks.git
2. open a new project and select the kvrocks dir

**Comment by tenghuanhe** [comment]
It works, thanks a lot @git-hulk
